package.json

```diff
@@ -2,10 +2,9 @@
     "name": "ghost-advisory-theme",
     "description": "Custom-built Ghost theme for Advisory SG",
     "demo": "https://advisory.sg",
-    "version": "1.0.0",
+    "version": "4.4.0",
```

Comment:
It's probably not necessary to update this version: it refers to the version of our theme package, not the version of Ghost. As a side note, maybe we might want to change it to 0.0.0 or 1.0.0-beta to indicate that it's not really done/production-ready.

Reply:
I think let's change to 0.0.0.